pkg/cluster/k8sres.go

```diff
@@ -437,9 +457,7 @@ func generatePodTemplate(
 		addShmVolume(&podSpec)
 	}
 
-	if nodeAffinity != nil {
-		podSpec.Affinity = nodeAffinity
```

Comment:
this changes previous behaviour where node affinity was set when node readiness labels are configured in operator configuration

Reply:
I've tried to respect the `node_readiness_label` with the pod anti affinity, but the operator crashed with nil pointer.

Reply:
@redbaron I've added the `node_readiness_label` to the pod anti affinity.